config/default.yaml

```diff
@@ -14,6 +14,10 @@ aws: {
     bucket: '',
     s3ForcePathStyle: false,
 }
+redis: {
+    host: 'development-pub-async-redis-3dqynz.serverless.euw2.cache.amazonaws.com',
```

Comment:
Let's change the default to something fake maybe?  Feels wrong to reference a certain environment by default.  Let's move this to `config/development.yaml`.